package.json

```diff
@@ -26,12 +26,12 @@
     "release": "aegir release",
     "build": "aegir build",
     "test": "aegir test",
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser"
+    "test:node": "aegir test --target node"
   },
   "dependencies": {
     "ioredis": "^4.19.2",
     "ipaddr.js": "^2.0.0",
+    "lodash.flatten": "^4.4.0",
     "winston": "^3.3.3"
```

Comment:
would be better to use Array.flat and consider pino instead of winston

Reply:
The linter complains because Array.flat was not available on Node 10. I could not change the engines to Node 12.

Thanks for the pino suggestion!

Reply:
why are you supporting node10 ?

Reply:
![image](https://user-images.githubusercontent.com/5447088/101893971-62e4d400-3b9d-11eb-8a7d-32314d72dfc0.png)

Apparently it's the TS checker. The linter is Ok.

Reply:
wait why are flattening that method returns an object not a array

Reply:
> why are you supporting node10 ?

@hugomrdias every time I change the engines in package.json to:

```
  "engines": {
    "node": ">=12.0.0"
  },
``` 

I get:

```
./node_modules/.bin/aegir lint-package-json

./package.json
✖ valid-values-engines - node: engines - Invalid value for engines
1 error
0 warnings
Command failed with exit code 2: npmPkgJsonLint /Users/henriquedias/Code/testground/sdk-js/package.json -c /Users/henriquedias/Code/testground/sdk-js/node_modules/aegir/src/config/.npmpackagejsonlintrc.json
```

Then I saw [this](https://github.com/ipfs/aegir/blob/master/src/config/.npmpackagejsonlintrc.json#L35-L40) and assumed we had to support Node 10.

Reply:
ah got it i will fix that, sorry my bad, that rule is dumb